corehq/apps/app_manager/views/forms.py

```diff
@@ -882,3 +886,15 @@ def _get_case_references(data):
         return references
     except Exception:
         raise ValueError("bad case references data: {!r}".format(refs))
+
+
+class FormHasSubmissionsView(LoginAndDomainMixin, View):
+    urlname = 'form_has_submissions'
+
+    def get(self, request, domain, app_id, form_unique_id):
+        app = get_app(domain, app_id)
+        form = app.get_form(form_unique_id)
+        form_has_submissions = FormES().domain(domain).app([app_id]).xmlns([form.xmlns]).count() != 0
```

Comment:
I'm not sure how ES works, but on SQL a count is much less efficient than a query with LIMIT 1 (and no ordering) or an EXISTS query. Does a count on ES do some type of aggregation that looks at every form or is it a quick lookup?

Reply:
Good to know.  I'll stick with the code as-is for now.